marketplace/wpp_templates/tests/test_requests.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+from unittest.mock import patch
+
+from marketplace.core.tests.base import FakeRequestsResponse
+
+from ..requests import TemplateMessageRequest
+from marketplace.core.types.channels.whatsapp_base.exceptions import FacebookApiException
+
+
+class TemplateMessageRequestTestCase(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.template_message_request = TemplateMessageRequest("32153243223")
+
+    @patch("requests.post")
+    def test_create_template_message_error(self, mock):
+        fake_response = FakeRequestsResponse(data={})
+        fake_response.status_code = 400
+
+        mock.side_effect = [fake_response]
```

Comment:
Since you only have one return, you don't need to use the `side_effect`. use the property `return_value`
line:
- [31](https://github.com/weni-ai/weni-integrations-engine/pull/167/files#diff-1e114a61f0e54edeca153f2505619fb956caf888992a28a0dfed3075debc94efR31)